server/http.go

```diff
@@ -349,10 +571,69 @@ func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID strin
 	if appErr = p.storeMeetingPostID(meetingID, createdPost.Id); appErr != nil {
 		p.API.LogDebug("failed to store post id", "err", appErr)
 	}
-
 	return nil
 }
 
+func (p *Plugin) askUserForMeetingPreference(userID, channelID string) {
```

Comment:
Is this specifically when the user is wanting to create a meeting? That should be reflected in the method name. Something like `askPreferenceForThisMeeting`